x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_list_page/index.tsx

```diff
@@ -178,11 +178,17 @@ export const AgentListPage: React.FunctionComponent<{}> = () => {
     });
   }, [search, selectedAgentPolicies, selectedStatus, selectedTags]);
 
+  // filters kuery for all tags
```

Comment:
Nit: this is not for filters :) The variable name is clear, so feel free to even remove the comment.